tests/click_tests/reset.py

```diff
@@ -0,0 +1,72 @@
+import shamir_mnemonic as shamir
+
+from trezorlib import messages
+
+from .. import buttons
+
+
+def confirm_wait(debug, startswith):
+    layout = debug.wait_layout()
+    assert layout.text.startswith(startswith)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def confirm_read(debug, startswith):
+    layout = debug.read_layout()
+    assert layout.text.startswith(startswith)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def set_selection(debug, button, diff):
+    layout = debug.read_layout()
+    assert layout.text.startswith("Slip39NumInput")
+    for _ in range(diff):
+        debug.click(button, wait=False)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def words_read(debug, is_advanced=False):
+    # read words
+    words = []
+    layout = debug.read_layout()
+    if is_advanced:
+        assert layout.text.startswith("Group")
+    else:
+        assert layout.text.startswith("Recovery share")
+    for i in range(6):
+        layout = debug.read_layout()
```

Comment:
I think this is unused?

Reply:
we need to call layout.read_layout() to "refresh" the state so we get the next page. at least that's how it seems to work. @matejcik ?

Reply:
but I removed the assignment since we don't use that

Reply:
That's probably correct, but what I meant is that you do not have to assign it to `layout`.

Reply:
You only need to call `read_layout` if you want to use its result. There are no side-effects.
(the result of `read_layout` is a snapshot at the time `read_layout` was called, so you need to call it again to get a new snapshot)

(`wait_layout` on the other hand blocks the test until the layout changes)